ndmg/scripts/multigraph_pipeline.py

```diff
@@ -43,7 +43,7 @@ def multigraphs(fibers, labels, outdir, gformat='gpickle'):
                   stdout=PIPE, stderr=PIPE, shell=True)
 
     # Create names of files to be produced
-    graphs = [op.join(outdir, "graphs", x, "{}_{}.{}".format(base, x, gformat))
+    graphs = [outdir + "/graphs/" + x + '/' + base + "_" + x + ".graphml"
```

Comment:
Are we only supporting graphml, now? I thought in graph.py, above, that you were only supporting weighted edgelist. Make sure this is consistent.

Reply:
This comment needs addressing, @ebridge2 . It's in one other place as well

Reply:
just gonna trash this; this is just a utility i use sometimes